downstream/assemblies/aap-on-azure/assembly-aap-azure-network-peering.adoc

```diff
@@ -8,22 +8,35 @@ ifdef::context[:parent-context: {context}]
 // [role="_abstract"]
 // You can use these instructions to install 
 
-Ansible Automation Platform on Azure is deployed into an independent managed resource group with its own Azure virtual network (VNet). 
+{AAPonAzureNameShort} is deployed into an independent managed resource group with its own Azure virtual network (VNet). 
 
-When initially deployed, Ansible Automation Platform on Azure’s VNet can only send requests to external networks through the public internet.
+When initially deployed, {AAPonAzureNameShort}’s VNet can only send requests to external networks through the public internet.
+
+To enable {AAPonAzureNameShort} to access resources in an internet-gapped deployment, when access to resources has to happen over private networks, you must configure Azure network peering between your private virtual networks and {AAPonAzureName}’s managed application VNet.
```

Comment:
Is internet-gapped the same as air-gapped?

Reply:
@ianf77 
It's not the same as air-gapped. Scott says it means that external requests via public internet are not permitted, but they are permitted over private networking.